manifest.go

```diff
@@ -0,0 +1,81 @@
+package feedx
+
+import (
+	"context"
+	"errors"
+	"strconv"
+	"strings"
+
+	"github.com/bsm/bfs"
+)
+
+// Manifest holds the current feed status.
+// the current manifest is consumed before each push and a new manifest written after each push.
+type Manifest struct {
```

Comment:
this should be in an internal package or private. we don't want to expose that in the public API

Reply:
since you are using core types, internal package won't work. please make it private. if you wanted to test something, expose it via `_ext_test.go` method